src/dialog/overflow-menu/overflow-menu.component.ts

```diff
@@ -32,6 +32,7 @@ import { OverflowMenuDirective } from "./overflow-menu.directive";
 		<div
 			[ibmOverflowMenu]="options"
 			[ngClass]="{'bx--overflow-menu--open': open}"
+			[className]="'bx--overflow-menu ' + triggerClass"
```

Comment:
The `bx--overflow-menu` should go in the normal `class` attribute, or the `[ngClass]` ... Then we can just pass `triggerClass` to the `[className]`

Reply:
for some reason that creates the property `triggerClass` on the component and sets it to the value of trigger input which causes the icon to be misplaced. An alternative that works and I think looks better is `class="bx--overflow-menu {{triggerClass}}"`

Reply:
Odd 🤔 Yeah I think I prefer `class="bx--overflow-menu {{triggerClass}}"` overall